src/master/integration/use-cases.md

```diff
@@ -0,0 +1,13 @@
+# Integration between immudb and other data technologies
+
+## Use cases
+
+immudb is designed to work with immutable data, using the relational model or the key/value model. For this reason it’s logical to use it as a primary way to persist the valuable and critical data.
+
+Not all data is valuable or critical. Non-critical data include caches, star schemas that can always be rebuilt, and time series data that will only be preserved for one week or two. This kind of data should not be persisted in immudb: for each of them, more suitable technologies can be used.
+
+While different data sets have different requirements and need to be stored in different technologies, sometimes they need to be used together. That’s the case, for example, when a data warehouse is built from heterogeneous sources. It is natural to move data from immudb to other relational databases to allow for further analyses.
```

Comment:
Good point, I will also add Data Lakes as they might be useful for Data Science use cases. Just a doubt: isn't the natural way to move data from immudb to Datawarehouse/Data Lake? Instead of moving to RDMS.

Reply:
I was thinking about cases when the RDBMS is the data warehouse: it happens with both PostgreSQL and MariaDB, and it's the intended use case for some Postgres derivatives. But maybe it's not clear. Plus, I realise only now that the verb "move" is wrong here (data won't be moved away from immudb). If I reword it this way will it sound better?

> While different data sets have different requirements and need to be stored in different technologies, sometimes they need to be used together. That’s the case, for example, when a data warehouse or a data lake are built from heterogeneous sources. Data will need to be copied from immudb to the data warehouse or data lake.

Reply:
Yes, it makes sense. Even so, with this title "Integration between immudb and other data technologies", I would mention Datawarehouse and  Data Lake - or restrict the topic to MariaDB/Postgres.

Reply:
Ok - I fixed it and made sure to mention data lakes